### ISSUE TITLE
Update "The alt="" attribute of an <img> is empty - is this a fail?"

### ISSUE THREAD

**Issue by Jefferydo** [cenw-wscoe/WCAG-in-Plain-Language#58]
The opening line of the solution is "If the image is decorative, it's ok and actually expected of it to be empty."

Mention "redundant" images and the effect on screen readers. Something like this:

"If the image is decorative or its meaning is redundant (already communicated in the text), the alt attribute is actually supposed to be empty. Screen readers will ignore an image with an empty alt attribute (alt="")."

Course page on [Decorative or redundant images](https://bati-itao.github.io/learning/esdc-self-paced-web-accessibility-course/module5/decorative-redundant-images.html).

Edit: Same with the next question, "An <img> lacks an alt="" attribute - is this a fail?". Mention "redundant" images.
